APKBUILD

```diff
@@ -28,6 +29,7 @@ package() {
   rm -rf "$pkgdir"/usr/glibc-compat/share
   rm -rf "$pkgdir"/usr/glibc-compat/var
   ln -s /usr/glibc-compat/lib/ld-linux-x86-64.so.2 ${pkgdir}/lib/ld-linux-x86-64.so.2
+  ln -s /usr/glibc-compat/lib/ld-linux-x86-64.so.2 ${pkgdir}/lib64/ld-linux-x86-64.so.2
```

Comment:
If it includes this file, then it should say `conflicts="libc6-compat"`; which makes it almost (everything except the `provides=` line) equivalent to https://github.com/sgerrand/alpine-pkg-glibc/pull/179/commits/edcbfe06d45cc2c2164890b1903bf58564a40567 from my PR.

If you wanted to take just one of my commits, I wouldn't be upset by that.

Reply:
Agreed, that's an important change.